pkg/storage/etcd/etcd_helper.go

```diff
@@ -101,7 +119,14 @@ func (h *etcdHelper) Create(key string, obj, out runtime.Object, ttl uint64) err
 	}
 
 	startTime := time.Now()
-	response, err := h.client.Create(key, string(data), ttl)
+
+	// TODO: With new api
+	keysAPI := etcd.NewKeysAPI(h.client)
+	opts := etcd.CreateInOrderOptions{
+		TTL: time.Duration(ttl),
+	}
+
+	response, err := keysAPI.CreateInOrder(ctx, key, string(data), &opts)
```

Comment:
It should use `Set(ctx context.Context, key, value string, opts *SetOptions) (*Response, error)` and set `PrevNoExist`. CreateInOrder is used to create in-order keys in one directory and user cannot specify key name.